src/components/layoutButton/LayoutChooser.styl

```diff
@@ -1,5 +1,34 @@
 $borderColor = rgba(77, 99, 110, 0.81)
 
+.pull-left
+  float: left!important;
+
+.dropdown-menu
+  position: absolute;
+  top: 100%;
+  left: 0;
+  z-index: 1000;
+  display: none;
+  float: left;
+  min-width: 160px;
+  padding: 5px 0;
+  margin: 2px 0 0;
+  font-size: 14px;
+  text-align: left;
+  list-style: none;
+  background-color: #fff;
+  -webkit-background-clip: padding-box;
+  background-clip: padding-box;
+  border: 1px solid #ccc;
+  border: 1px solid rgba(0,0,0,.15);
+  border-radius: 4px;
+  -webkit-box-shadow: 0 6px 12px rgba(0,0,0,.175);
+  box-shadow: 0 6px 12px rgba(0,0,0,.175);
+
+.btn-group, .btn-group-vertical
+  position: relative;
+  display: inline-block;
+  vertical-align: middle;
```

Comment:
Similarly, `.btn-group` may be defined "higher up", or included in a button specific file